src/Components/Blazor/WebAssembly.Authentication/src/Models/RemoteAuthenticationResult.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// Represents the result of an authentication operation.
+    /// </summary>
+    /// <typeparam name="TState">The type of the preserved state during the authentication operation.</typeparam>
+    public class RemoteAuthenticationResult<TState> where TState : class
+    {
+        /// <summary>
+        /// Gets or sets the status of the authentication operation. The status can be one of <see cref="RemoteAuthenticationStatus"/>.
+        /// </summary>
+        public string Status { get; set; }
```

Comment:
As commented elsewhere, it would be more natural for this to be an enum type. Is there any way of achieving that?